app/models/dashboards/overview.rb

```diff
@@ -5,6 +5,12 @@
 ###
 
 class Dashboards::Overview < Dashboards::Base
+  def initialize(filter)
+    @start_date = filter.start
+    @end_date = filter.end
```

Comment:
This doesn't matter, but I wonder if delegating off to the filter would be nicer.  It would mean updating a bunch of things that take `start_date:, end_date:`, but if they took a range, you could just pass them `filter.range`, which seems way cleaner.